Frontend/src/components/CodeSmellList.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import Card from "react-bootstrap/Card";
+import InteractiveList from "./InteractiveList";
+import { Settings } from "../config/settings";
+
+const CodeSmellList = () => {
+    const items = [
+        { id: 1, label: "Option 1" },
+        { id: 2, label: "Option 2" },
+        { id: 3, label: "Option 3" },
+    ];
+
+    const [selectedItemId, setSelectedItemId] = useState<string | number | undefined>();
+
+    const handleItemClick = (itemId: string | number) => {
+        setSelectedItemId(itemId);
+        // Do something with the selected item ID logging for now
+        console.log();
+    };
+
+    return (
+        <Card border="dark" bg={Settings.COMPONENT_VARIANT} style={{ width: "36rem" }}>
+            <Card.Body>
+                <Card.Title>Code Smells</Card.Title>
+                {/* <Card.Subtitle className="mb-2 text-muted">
```

Comment:
git ma historię, nie musimy trzymać wykomentowanego kodu

Reply:
Nie do końca się z tym zgadzam. Nie mamy tego kodu aż tyle żeby pojedyncze komentarze kodu bardzo zaśmiecały. Nie wiemy czy to się przyda, a historia gita przydatna jest dopiero wtedy jak wiesz że coś już w kodzie było. Usunołem zakomentowany kod z `index.tsx` ale zostawilem  w `CodeSmellList.tsx` bo to mój kod jest tam zakomentowany